e2e/fixtures.ts

```diff
@@ -0,0 +1,13 @@
+import { Locator, test as base } from '@playwright/test'
+
+interface Fixtures {
+  componentElement: Locator
+}
+
+export const test = base.extend<Fixtures>({
+  componentElement: async ({ page }, use) => {
+    await use(page.locator('#root > *:first-child'))
+  },
+})
+
+export { expect } from '@playwright/test'
```

Comment:
I wonder if we would need the to this 

```ts
export * from '@playwright/test'
export const test = base.extend<Fixtures>({
  componentElement: async ({ page }, use) => {
  //code
  },
})
```
so that in our test if we had to we can import the types

Reply:
Yeah was thinking about the types. Gave it a go but it seems ESLint can't resolve the types unless they're a named export, and I don't think we want name all of them. I think we can say types can continue to be imported from `@playwright/test`.

Reply:
Scratch that, I think it was just a caching problem in the IDE :man_facepalming: